testapk.txt

```diff
@@ -0,0 +1,16 @@
+WARNING: Ignoring APKINDEX.00740ba1.tar.gz: No such file or directory
```

Comment:
In PR #7 I started down the path of having tests that actually spun up docker containers and validated it did what it should. It's cool that this file was added and the comment in the PR on how to use it but realistically after this PR is merged this file is just a thing that is not used other than in manual testing (maybe).

Reply:
I like where you are going, it would be nice to replace these with real tests!